### DISCUSSION TITLE
Neovim not working with network shares

### DISCUSSION THREAD

**Discussion by tomasalves8** [community/community#42231] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I'm using Neovim (Windows) to work on a project that is being shared on a network.

No Suggestions show up and when I enter the command ':Copilot'. I get the following error:
`Request getPanelCompletions failed with message: [UriError]: If a URI does not contain an authority component, then the path cannot begin with two slash characters ("//")`

It seems there is a problem with the UNC Path. \\\192.168.1.3\FolderName

**Comment by getlost01** [comment]
It looks like you are experiencing an error when using the Copilot plugin for Neovim on Windows. The error message mentions a problem with the UNC path (\192.168.1.3\FolderName), which is a type of network path used to access shared resources on a network.

There are a few things you can try to resolve this issue:
1. Check the network connection: Make sure that your computer is connected to the network and that you have access to the shared resource.
2. Check the UNC path: Make sure that the UNC path is correct and that the shared resource actually exists at that location. You can try accessing the shared resource using Windows Explorer to see if it is available.
3. Check the Copilot plugin: Make sure that the Copilot plugin is installed and configured correctly. You may need to consult the plugin documentation or seek assistance from the plugin developers for troubleshooting.
4. Use a different network path: If the UNC path is not working, you can try using a different network path, such as a mapped drive letter or a URL. For example, you could map a drive letter to the shared resource and use that drive letter in your Neovim project path.

I hope these suggestions help you resolve the issue with the UNC path and the Copilot plugin in Neovim.

**Reply by tomasalves8** [reply]
The network share works fine.

**Reply by tomasalves8** [reply]
Did you use ChatGPT to get that answer?

**Reply by getlost01** [reply]
Yes, I do, because it gives me the perfect answer and also improves my knowledge though.

**Reply by tomasalves8** [reply]
In this case it wasn't perfect.

**Reply by MrDwarf7** [reply]
Also had a sneaking suspicion that this was a Jippity answer, amusing that it's spot-able so easily

**Comment by tpope** [comment, marked as answer]
This was fixed in Copilot.vim 1.33.0.
